netbox/overlays/ocp-staging/kustomization.yaml

```
@@ -6,6 +6,7 @@ resources:
 - ../ocp-prod
 
 patches:
+- path: deployments/netbox_config_patch.yaml
 - path: postgresclusters/netbox_remote-backups_patch.yaml
 - path: externalsecrets/netbox-backup_patch.yaml
 - path: externalsecrets/oauth2-secret_patch.yaml
```

Rather than inserting the same image name three times in the deployment patch, you can handle it in your `kustomization.yaml` in one place:

```suggestion
- path: externalsecrets/oauth2-secret_patch.yaml

images:
  - name: quay.io/netboxcommunity/netbox
    newName: quay.io/hakasapl/netbox-moc-plugins
    newTag: latest
```